CHANGELOG.rst

```diff
@@ -2,6 +2,13 @@
 Changes for ya16sdb
 ===================
 
+0.8
```

Comment:
Looks like 0.7 tag was never actually applied, so perhaps should just merge these changes in with the 0.7 section and plan for that tag to refer to this release.